docs/glossary.md

```diff
@@ -0,0 +1,84 @@
+**Aliases**: CSV(s)
+
+### CatalogSources
+
+**Definition**: The CatalogSource represents a repository of bundles, which are collections of resources that must contain [CSVs](#ClusterServiceVersion), [CRDs](#CustomResourceDefinitions), and package definitions. There are multiple implementations of a CatalogSource backend, the current recommendation is to use a [registry image](#Index).
+**Aliases**: CatSrc(s)
+
+### InstallPlans
+
+**Definition**: The InstallPlan defines a set of resources to be created in order to install or upgrade to a specific version of [CSV](#ClusterServiceVersion). 
+**Aliases**: IP(s)
+
+### OperatorGroups
+
+**Definition**: The OperatorGroup selects a set of target namespaces in which to generate required RBAC access for its member Operators.
+**Aliases**: OG(s)
+
+### OperatorSources
+
+**Definition**: The OperatorSources are external app registries that contains a catalog of operators. Applying an OperatorSource to a cluster makes the operators in that OperatorSource available for installation in that cluster.
+**Aliases**: OpSrc(s)
+
+### Subscriptions
+
+**Definition**: The Subscription defines the channel and the source of Operator updates and is used to keep the CSV updated by binding with a channel in a package. 
+**Aliases**: Subs(s)
+
+## OLM Concepts
+
+### Bundle
+
+**Definition**: A collection of Operator [CSV](#ClusterServiceVersion), manifests, and metadata which together form a unique version of an Operator that can be installed onto the cluster. 
+**Aliases**:
+
+### Bundle Image
+
+**Definition**: An image of a bundle is built from operator manifests and contains exactly one [bundle](#Bundle). The bundle images are stored and distributed by app registries such as Quay.io or DockerHub.
+**Aliases**:
+
+### Channel
+
+**Definition**: The channel defines a stream of updates for an operator and is used to roll out updates for subscribers. The head points at the lastest version of that channel. For example, a stable channel would have all stable versions of an operator arranged from the eariliest to the lastest. An operator can have several channels, and a subscription binding to a certain channel would only look for updates in that channel.
+**Aliases**:
+
+### Catalog Image (OPM Index)
+
+**Definition**: A catalog image is an image that contains Operator bundles that are stored in a database. These are then exposed over gRPC to the OLM Operator to allow users to query and install new Operators onto the cluster. 
+**Aliases**: 
+
+### Dependency
+
+**Definition**: An Operator may have a dependency on another Operator being present in the cluster. For example, the Vault Operator has a dependency on the Etcd Operator for its data persistence layer. OLM resolves these dependencies by ensuring all specified versions of Operators and CRDs are installed on the cluster during the installation phase. This dependency is based on API and not related to [packages](#Packages)/[bundles](#Bundles).
```

Comment:
We might want to be more specific here and rather than just say `This dependency is based on API` we should say `This dependency is resolved by finding and installing an Operator in a Catalog that satisfies the required CRD API`